test/stage2/bondingEvent.js

```diff
@@ -469,12 +475,12 @@ describe('BondingEvent', async () => {
       );
       const positions = await BondingEvent.getPositions();
       expect(positions).to.be.length(2);
-      await swap(USDC, SEuro, etherBalances['125K'].mul(5));
 
       await expect(BondingEvent.clearPositionAndBurn(positions[0].tokenId)).to.be.revertedWith('err-no-wallet-assigned');
 
       await BondingEvent.setExcessCollateralWallet(wallet.address);
 
+      // const collect = BondingEvent.clearPositionAndBurn(positions[0].tokenId);
```

Comment:
dead comment

Reply:
sure, you can push to master 👍

Reply:
can i? sadly i don't think i can 😢

Reply:
no, I disabled that for all of us. next time you open a PR you can remove it, or if you see some other cleanup to do you can open a PR with the fix too 😄